app/src/main/res/menu/download_log.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:custom="http://schemas.android.com/apk/res-auto">
```

Comment:
Please use `xmlns:app` instead to be more consistent with the rest of the code base.